site/content/docs/main/config/request-routing.md

```diff
@@ -454,7 +454,7 @@ spec:
       strategy: RequestHash
       requestHashPolicies:
       - queryParameterHashOptions:
-          prameterName: param1
+          parameterName: param1
```

Comment:
As noted earlier for a similar documentation item, this typo only appears in docs and there's nothing to indicate it's intentional.